enterprise/admin/forms.py

```diff
@@ -46,6 +46,15 @@ class ManageLearnersForm(forms.Form):
             ("honor", _("Honor")),
         ],
     )
+    notify_on_enrollment = forms.ChoiceField(
+        label=_("Notify learners of enrollment"),
+        choices=[
+            ("email", _("Email")),
```

Comment:
Since it appears that we are describing the notification mechanism with these choices, we might instead say "by_email", instead of just "email".